doc/source/gotcha.rst

```diff
@@ -455,7 +307,7 @@ where the data copying occurs.
 See `this link <http://stackoverflow.com/questions/13592618/python-pandas-dataframe-thread-safe>`__
 for more information.
 
-.. _html-gotchas:
+.. _faq.html:
 
 HTML Table Parsing
 ------------------
```

Comment:
We could maybe also move this to the actual html docs? (it does not seem to fit very well in the gotchas/faq section)